### PR TITLE
[Hub Generated] Review request for Microsoft.Consumption to add version stable/2019-10-01

### PR DESCRIPTION
**Pull request by sashahdo**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/sashahdo/azure-rest-api-specs/dev-consumption-Microsoft.Consumption-2019-10-01...master/).

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-10-01/consumption.json [hunk 1]
```diff
@@ -3336,13 +3336,13 @@
           }
         },
         "timePeriod": {
-          "description": "Has start and end date of the budget. The start date must be first of the month and should be less than the end date. Budget start date must be on or after June 1, 2017. Future start date should not be more than three months. Past start date should  be selected within the timegrain period. There are no restrictions on the end date.",
+          "description": "Has start and end date of the budget. The start date must be first of the month and should be less than the end date. Budget start date must be on or after June 1, 2017. Future start date should not be more than twelve months. Past start date should  be selected within the timegrain period. There are no restrictions on the end date.",
           "$ref": "#/definitions/BudgetTimePeriod"
         },
-        "filters": {
+        "filter": {
```
Comment (weidongxu-microsoft) [inline specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-10-01/consumption.json:3342]: This is stable version. I think changes like these are breaking change, and should not happen to stable.

Comment (sashahdo) [reply specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-10-01/consumption.json:3342]: "filters" is not how the api is currently implemented. This is from an old version.
I'm updating this to the correct current implementation.

**Review by weidongxu-microsoft — APPROVED** [review]

**Review by KrisBash — APPROVED** [review]
My understanding is that this is a bugfix to match swagger to service behavior. if so, signing off. If that is not the case, please move breaking change to new API version

File: specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-10-01/consumption.json [hunk 2]
```diff
@@ -3384,51 +3384,64 @@
...
+          "type": "string"
+        },
+        "operator": {
+          "description": "The operator to use for comparison.",
+          "type": "string",
+          "enum": [
+            "In"
+          ],
+          "x-ms-enum": {
+            "name": "OperatorType",
```
Comment (weidongxu-microsoft) [inline specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-10-01/consumption.json:3426]: Please check the x-ms-enum name, it conflicts with this one https://github.com/Azure/azure-rest-api-specs/blob/e8fd0dcdc31abae062b809741cffc0a9ac9edf65/specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-10-01/consumption.json#L3471-L3479

Comment (sashahdo) [reply specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-10-01/consumption.json]: thanks for catching that. I've fixed it in the newest commit

**Comment by weidongxu-microsoft** [comment]
@akning-ms 

Hello Arthur, would you help merge this PR?

The BreakingChange is to fix the swagger to reflect correct service behavior.

> "filters" is not how the api is currently implemented. This is from an old version.
I'm updating this to the correct current implementation.

**Pull request merged**